state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -285,7 +287,9 @@ pub mod pallet {
 			source_asset: Asset,
 			deposit_amount: AssetAmount,
 			destination_asset: Asset,
+			#[deprecated(note = "Use swap_output instead")]
 			egress_amount: AssetAmount,
```

Comment:
if we rename this, would it make sense to rename the `deposit_amount` too? or is `deposit_amount` the appropriate term for the amount with the ingress fee deducted?

Reply:
Yeah swap_input probably makes more sense here. 

I think terminology should be something like this:

`deposit_amount` (`ingress_amount` ?): the amount sent by the user
`swap_input`: the amount that is actually swapped (ie. deposit amount minus broker fee and gas fees)
`swap_output`: the result of the swap (net liquidity and network fee)
`egress_amount`: the amount sent to the user (swap output minus egress fee)